test-case-reporting/app.ts

```diff
@@ -87,7 +87,44 @@ function lowerCaseStatus({status, ...rest}: TestRun): any {
 
 app.use(express.static('static/css'));
 
-app.get(['/', '/builds'], async (req, res) => {
+app.get('/', async (req, res) => {
+  const {json} = req.query;
+
+  try {
+    const pageInfo: PageInfo = {limit: 10, offset: 0};
+    const builds = await record.getRecentBuilds(pageInfo);
+
+    if (json) {
+      // Leaving this in the base path in case there is some automation using this.
+      res.json({builds});
```

Comment:
Context: This project was built to mainly be a backend/data-collection. In order for Ricardo to have something to show, the front-end was thrown together, with all endpoints supporting JSON, with the expectation (hope?) that down the line it would get a proper frontend, such as the release dashboard, which would poll these API endpoints for JSON directly